model/user.py

```diff
@@ -1 +1,62 @@
 """Data model to represent an individual user."""
+from permissions import Permissions
+
+
+class User:
```

Comment:
@rwblickhan Does the user need a uuid as well?

Reply:
I’d say just using Slack ID is probably good enough, since those should be unique and we want to uniquely tie our users to Slack users. But I’m open to other ideas as well!